src/micromamba/main.cpp

```diff
@@ -684,6 +687,13 @@ install_specs(const std::vector<std::string>& specs, bool create_env = false, in
     trans.execute(prefix_data, pkgs_dirs);
 }
 
+void
+update_specs(const std::vector<std::string>& specs)
+{
+    std::cout << specs[0] << std::endl;
```

Comment:
?

Reply:
sorry for that it's wip :)
I should have created a draft PR

Reply:
no worries.